lib/factory_bot/linter.rb

```diff
@@ -52,6 +60,15 @@ def location
       end
     end
 
+    def lint(factory)
+      errors = lint_factory(factory)
+      if @traits
+        errors | lint_traits(factory)
```

Comment:
The `|` operator always throws me - is this intended or would `||` work?

Reply:
It was there before (as `|=`) and I didn't think to much about it. I will figure out whether it is necessary.